app/rental_routes.py

```diff
@@ -0,0 +1,77 @@
+from app import db
+from app.models.customer import Customer
+from app.models.video import Video
+from app.models.rental import *
+from flask import Blueprint, request, make_response , jsonify
+import datetime as dt
+import os 
+
+rental_bp = Blueprint("rentals", __name__, url_prefix="/rentals")
+
+
+@rental_bp.route("/check-out", methods = ["POST"])
+def check_out_rental():
+    request_body = request.get_json()
+    try:
+        customer = Customer.query.get(request_body["customer_id"])
+        video = Video.query.get(request_body["video_id"])
+    except:
+        return make_response("invalid input", 400)
+    if customer is None or video is None:
+        return make_response("", 404)
+    if count_a_videos_inventory(video.id, video.total_inventory) == 0:
+        return make_response({"message":"Could not perform checkout"}, 400)
+
+    new_rental = Rental(
+        customer = customer.id,
+        video = video.id,
+        videos_checked_out_count = len(query_customers_videos(customer.id)) + 1,
+        available_inventory = count_a_videos_inventory(video.id, video.total_inventory) - 1,
```

Comment:
while this works, it definitely falls to the programmer to remember to subtract and add these values every time. One thing to consider next time would be to leave these attributes off the model and calculate them through helper functions or your model methods instead. 

That way, all you have to do is make some queries instead of also keeping track of a new value